src/navigation/Navigation.js

```diff
@@ -118,7 +119,10 @@ function getActiveRouteName(navigationState) {
  */
 function handleAction(name, params, replace = false) {
   if (!TopLevelNavigationRef) return;
-  const action = StackActions[replace ? 'replace' : 'push'](name, params);
+  const action = (replace ? StackActions.push : CommonActions.navigate)(
```

Comment:
I totally support the change from StackActions.push to CommonActions.navigate.

However, originally when replace was set to true we were calling `StackActions.replace` but with this change it's calling `StackActions.push` instead.

Just confirming that was intended and not a mistake.